### ISSUE TITLE
[support] Error running Docker image

### ISSUE THREAD

**Issue by romaricdrigon** [rectorphp/rector#1159]
Hello,

I've been trying to use Rector to migrate some Symfony 2.8 application, but I couldn't make it work any way.
Specifically, I tried to use the Docker image.
While running `docker run -v $(pwd):/project rector/rector:latest`, I'm getting this error:
```
Fatal error: Uncaught Symplify\PackageBuilder\Exception\Configuration\FileNotFoundException: File "/project/rector.yaml" not found in "/project/rector.yaml". in /rector/vendor/symplify/package-builder/src/Configuration/ConfigFileFinder.php:26
Stack trace:
#0 /rector/bin/container.php(15): Symplify\PackageBuilder\Configuration\ConfigFileFinder::detectFromInput('rector', Object(Symfony\Component\Console\Input\ArgvInput))
#1 /rector/bin/rector(21): require_once('/rector/bin/con...')
#2 {main}
  thrown in /rector/vendor/symplify/package-builder/src/Configuration/ConfigFileFinder.php on line 26
```

I created an empty `rector.yaml` file in my projet, it did not help. Any insight?
Thanks

**Comment by mrpixeldream** [comment]
Are you running the docker command from the root directory of your project where the config file lives as well?

**Comment by romaricdrigon** [comment]
Yes. The config file is empty though, from the documentation I don't know what to write inside.

**Comment by mrpixeldream** [comment]
I can not reproduce the issue locally. When a file called `rector.yaml` exists in the project root, the command can be executed successfully, even when said file is empty. Are you sure you didn't name it `rector.yml` (without "a") by accident? Even though it should work that way as well I think, it does not work with `yml` locally on my computer.

If that's not the case, I think we might need more information about your environment and/or the project structure.

**Comment by romaricdrigon** [comment]
Indeed now I don't get anymore the same error when running the Docker image with an empty `rector.yaml` file, maybe the code was modified?
The output now:
```
Rector No version set (parsed as 1.0.0)@
Config file: /project/rector.yaml


In RectorGuard.php line 26:
                                                    
  We need some rectors to run:                      
  * register them in rector.yaml under "services:"  
  * use "--level <level>"                           
  * or "--config <file>.yaml"                       
```

The way the Docker file is made I can't pass `--level` flag directly, is there any (undocumented) way to pass it through config file?

**Comment by romaricdrigon** [comment]
Update, I made it work using this command: 
`docker run -v $(pwd):/project rector/rector:latest bin/rector process /project --level xxx`

I think the default flags to rector are a little bit misleading, and the Docker part of the documentation is a little bit misleading. Giving this command as exmeple woudl be better IMO, I can open a quick PR:
`docker run -v $(pwd):/project rector/rector:latest bin/rector process /project/src --dry-run --level symfony40`

**Comment by mrpixeldream** [comment]
There's currently a PR open that lets you configure the options more easily: #1183  
I think this is gonna be merged soon. After that, we can update the documentation. A PR is always appreciated! :)

**Comment by JanMikes** [comment]
Hi! Agreed, documentation part for original use of Docker (before #1183) was a bit misleading. We will fix it soon.

I will just confirm what you have already mentioned above. You have to set volume your project files to `/project` dir in docker container and then analyse this folder, etc `/project/src`

**Comment by JanMikes** [comment]
Should be solved via #1183 - changed usage of rector in docker and clarified usage in the documentation.

@romaricdrigon please try if it works now for you and feels free to reopen if you need further support.
